src/config/environment.json

```diff
@@ -0,0 +1,24 @@
+{
+    "type": "firstDiscovery.server.environment.config",
+    "options": {
+        "gradeNames": ["gpii.firstDiscovery.server.configurator"],
+        "port": "{environment}.env.FIRST_DISCOVERY_SERVER_TCP_PORT",
+        "preferencesConfig": {
+            "securityServer": {
+                "port": "{environment}.env.GPII_OAUTH2_TCP_PORT",
```

Comment:
I worry that "environment" is a bit too generic of a context name to use here -- what do you think about use of a grade name such as "gpii.envVars"? Or perhaps "gpii.resolvers.env" - preparing for a world in which we have a larger family of these things.